terraform/cp4ba/variables.tf

```diff
@@ -70,36 +73,46 @@ variable "on_vpc" {
   description = "Select 'true' to install on a VPC cluster and it's using VPC Gen2. Note: CP4BA does not currently support VPC cluster."
 }
 
+variable "entitlement" {
```

Comment:
This should be defaulted to `cloud_pak`.  In fact, I wouldn't even have it here.  Force it in the module call in main.tf.

Reply:
Fixed